e2e/conftest.py

```diff
@@ -8,6 +9,15 @@
 from e2e.pages.maps.edit import MapEditPage
 from e2e.pages.maps.planting import MapPlantingPage
 
+load_dotenv()
+
+E2E_TIMEOUT: float = float(os.getenv("E2E_TIMEOUT", 30000))
+E2E_WIDTH: int = int(os.getenv("E2E_WIDTH", 1920))
+E2E_HEIGHT: int = int(os.getenv("E2E_HEIGHT", 1080))
```

Comment:
Please try to keep as simple as possible. If there is no need to have e.g. a setting differently locally and in deployment, we also don't need to have it configurable.

Reply:
Alright.